corehq/apps/auditcare/utils/export.py

```diff
@@ -70,6 +68,48 @@ def get_all_log_events(start_date=None, end_date=None):
     )
 
 
+def get_domain_first_access_times(domains, start_date=None, end_date=None):
+    """Query NavigationEventAudit events for _first event matching any of
+    `domains` within each authenticated session_.
+
+    NOTE: This function does _not_ query couch.
+
+    NOTE: This function may return multiple "access events" from the same
+          session (if multiple of `domains` were accessed in the same session).
+
+    Resulting SQL query:
+
+    ```sql
+    SELECT
+        MIN(event_date) AS access_time,
+        "user",
+        domain,
+        session_key
+    FROM auditcare_navigationeventaudit
+    WHERE (
+        domain IN ( {domains} )
+        AND event_date > {start_date}
+        AND event_date < {end_date}
```

Comment:
The `end_date` filter is inclusive
https://github.com/dimagi/commcare-hq/blob/5a0a6be2ff5851c830196f70504b826c88fd2011/corehq/apps/auditcare/utils/export.py#L149

```suggestion
        AND event_date <= {end_date}
```